projects/seronet-spawner/controllers/box_creator/utils/BoxType.cc

```diff
@@ -0,0 +1,14 @@
+#include "BoxType.hh"
+#include <algorithm>
+
+BoxType::BoxType(string proto, double *rotation, double *size) : proto(proto) {
+  for (int i = 0; i < 4; i++) {
+    this->rotation[i] = rotation[i];
+  }
+  for (int i = 0; i < 3; i++) {
+    this->size[i] = size[i];
+  }
```

Comment:
Coding style is wrong:
```suggestion
  for (int i = 0; i < 4; i++)
    this->rotation[i] = rotation[i];
  for (int i = 0; i < 3; i++)
    this->size[i] = size[i];
```